content/docs/zero/install.mdx

```diff
@@ -25,18 +25,21 @@ If you haven't signed up for Pomerium Zero, you can [**create an account**](http
 :::
 
 <Tabs>
-<TabItem value="linux" label="Linux">
+<TabItem value="kubernetes-helm" label="Kubernetes (Helm)">
 
-Install Pomerium with the following shell script:
+To install Pomerium with Helm, run the following command:
 
 ```bash
-curl https://console.pomerium.app/install.bash | env POMERIUM_ZERO_TOKEN=<CLUSTER_TOKEN> bash -s install
+helm install pomerium-zero oci://docker.io/pomerium/pomerium-zero \
+  --set pomeriumZeroToken=<CLUSTER_TOKEN> \
+  --namespace pomerium-zero \
+  --create-namespace
```

Comment:
can we make those two defaults so that only cluster token would be a required arg

Reply:
You're not supposed to put a namespace in the chart templates, since it's specified from this arg. I don't think helm will create the namespace unless you add --create-namespace either